trail/build.gradle

```diff
@@ -55,10 +55,13 @@ android {
 dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-  def playServicesVersion = '11.4.2'
+  def playServicesVersion = '11.8.0'
```

Comment:
Newest version is 15.0.1. Have you test 15.0.1 with this repo? @j05u3

Reply:
I haven't tested with 15.0.1 only with 11.8.0